### PR TITLE
chore: add license

### PR DESCRIPTION
**Pull request by Wackyator**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,6 +14,7 @@ categories = ["conda"]
 homepage = "https://github.com/mamba-org/rattler"
 repository = "https://github.com/mamba-org/rattler"
 license = "BSD-3-Clause"
+license-files = { paths = ["LICENSE"] }
```
Comment (baszalmstra) [inline Cargo.toml:17]: This one is not needed if license has been specified

File: pixi.toml [hunk 2]
```diff
@@ -2,9 +2,16 @@
...
+authors = [
+    "Wolf Vollprecht <wolf@prefix.dev>",
+    "Bas Zalmstra <bas@prefix.dev>",
+    "Tim de Jager <tim@prefix.dev>",
+    "Ruben Arts <ruben@prefix.dev>",
+]
 channels = ["conda-forge"]
 platforms = ["linux-64", "win-64", "osx-64", "osx-arm64"]
+license = "BSD-3-Clause"
+license-files = { paths = ["LICENSE"] }
```
Comment (baszalmstra) [inline pixi.toml:14]: This one is also not needed if license is specified.

File: py-rattler/Cargo.toml [hunk 3]
```diff
@@ -2,15 +2,21 @@
 name = "py-rattler"
 version = "0.1.0"
 edition = "2021"
+license = "BSD-3-Clause"
+license-files = { paths = ["LICENSE"] }
```
Comment (baszalmstra) [inline py-rattler/Cargo.toml:6]: Same!

File: py-rattler/LICENSE [hunk 4]
```diff
@@ -0,0 +1,28 @@
+BSD 3-Clause License
```
Comment (baszalmstra) [inline py-rattler/LICENSE:1]: Maybe you can turn this into a symlink to the file at the root of the repo? So we dont duplicate this?

File: py-rattler/pixi.toml [hunk 5]
```diff
@@ -2,9 +2,14 @@
...
 description = "Add a short description here"
-authors = ["Bas Zalmstra <bas@prefix.dev>"]
+authors = [
+    "Bas Zalmstra <bas@prefix.dev>",
+    "Tarun Pratap Singh <pratapsinghtarun9@gmail.com>",
+]
 channels = ["conda-forge"]
 platforms = ["win-64", "linux-64", "osx-64", "osx-arm64"]
+license = "BSD-3-Clause"
+license-files = { paths = ["LICENSE"] }
```
Comment (baszalmstra) [inline py-rattler/pixi.toml:12]: Same

File: py-rattler/pyproject.toml [hunk 6]
```diff
@@ -6,13 +6,16 @@ build-backend = "maturin"
...
+    { name = "Bas Zalmstra", email = "zalmstra.bas@gmail.com" },
+    { name = "Tarun Pratap Singh", email = "pratapsinghtarun9@gmail.com" },
 ]
 classifiers = [
     "Programming Language :: Rust",
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
+license = "BSD-3-Clause"
+license-files = { paths = ["LICENSE"] }
```
Comment (baszalmstra) [inline py-rattler/pyproject.toml:18]: Not sure about pyproject but I assume either license or license-file is required.

File: Cargo.toml (hunk 1 above)
Comment (baszalmstra) [reply Cargo.toml:17]: see: https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields

Comment (Wackyator) [reply Cargo.toml:17]: i just realised, fixed it

File: py-rattler/LICENSE (hunk 4 above)
Comment (Wackyator) [reply py-rattler/LICENSE:1]: i could do this, but is there any possibility that py-rattler might me moved to its own repo?

Comment (baszalmstra) [reply py-rattler/LICENSE:1]: personally Id like to keep them together to make sure the two code bases stays as much in sync as possible.

**Pull request merged**